03-javascript-derive-computed-values-and-manage-side-effects-with-mobx-reactions/index.js

```diff
@@ -41,3 +40,4 @@ const App = observer(({ temperature }) => (
 const rootElement = document.getElementById("root");
 ReactDOM.render(<App temperature={t} />, rootElement);
 
+global.t = t; /* you can now use `t.unit = "K"` etc in the sandbox console
```

Comment:
Oh, didn't realize we could just set a global like this for the sandbox console. Nice addition!

Reply:
That's awesome! Thank you for showing me that.

Reply:
:)